### PR TITLE
JA: Translation for ja/beta/tutorials/eager/automatic_differentiation.ipynb

### PR DESCRIPTION
**Pull request by masa-ita**

### CODE REVIEW

**Comment by masa-ita** [comment]
As the 2.0.0rc0 was released, I moved site/ja/beta/tutorials/eager/automatic_differentiation.ipynb to site/ja/tutorials/customization/automatic_differenctiation.ipynb, and fixed links etc.

File: site/ja/tutorials/customization/automatic_differentiation.ipynb [hunk 1]
```diff
@@ -0,0 +1,364 @@
+{
```
Comment (AseiSugiyama) [inline site/ja/tutorials/customization/automatic_differentiation.ipynb]: <p>I guess <code>デフォルトでは</code> &nbsp;is better than <code>既定では</code>&nbsp;.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (AseiSugiyama) [inline site/ja/tutorials/customization/automatic_differentiation.ipynb]: <p><a href="https://docs.python.org/ja/3/library/stdtypes.html#context-manager-types" target="_blank">In Python documentation</a>, <code>Context Manager</code> is translated into <code>コンテキストマネージャ</code>. I guess <code>コンテキストマネージャ</code> is better than <code>コンテキストマネジャー</code> for consistency.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

**Comment by AseiSugiyama** [comment]
@masa-ita I reviewed your PR, and leave some comments. Please check it.

File: site/ja/tutorials/customization/automatic_differentiation.ipynb (hunk 1 above)
Comment (masa-ita) [reply site/ja/tutorials/customization/automatic_differentiation.ipynb]: Thank you．I've changed it.

Comment (masa-ita) [reply site/ja/tutorials/customization/automatic_differentiation.ipynb]: Thanks.  I did not notice this.  Fixed.

**Pull request merged**